hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -418,6 +478,12 @@ public SnapshotDiffResponse getSnapshotDiffReport(
               bucketName, fromSnapshotName, toSnapshotName, new ArrayList<>(),
               null),
           REJECTED, defaultWaitTime);
+    case CANCELED:
+      return new SnapshotDiffResponse(
+          new SnapshotDiffReportOzone(snapshotRoot.toString(), volumeName,
+              bucketName, fromSnapshotName, toSnapshotName, new ArrayList<>(),
+              null),
+          CANCELED, 0L, CancelStatus.CANCEL_SUCCESS);
```

Comment:
Unrelated to this PR but I think `SnapshotDiffReportOzone` could use a `Builder` subclass. cc @hemantk-12